wm_2dnav/config/costmap_common_params.yaml

```diff
@@ -1,11 +1,11 @@
 obstacle_range: 4.0
 raytrace_range: 5.0
 
-# footprint: [[front left corner], [front right corner], 
+# footprint: [[front left corner], [front right corner],
 #             [rear right corner], [rear left corner]]
 footprint: [[0.40, -0.33], [0.40, 0.33], [-0.40, 0.33], [-0.40, -0.33]]
 
-inflation_radius: 0.40
+inflation_radius: 0.10
```

Comment:
Are those change related to the new repo structure or have been introduced by mistake?